Utils/Dataflow/pyDKB/common/json_utils.py

```diff
@@ -30,10 +72,16 @@ def valueByKey(json_data, key):
 def nestedKeys(key):
     """ Transform STRING with nested keys into LIST.
 
-    Parameters:
+    **Parameters:**
         STRING key -- dot-separated list of nested keys.
-                      If a key contains dot itself, the key must be put between
-                      quotation marks.
+                      If a key contains dot itself, the key must be put
+                      between quotation marks.
+
+    **Examples:**
+        Transform STRING ``\'1.2.3'`` into LIST ``[\'1', \'2', \'3']``.
+
+        Transform STRING ``\'1.\"2.3".4'`` into
```

Comment:
Talking about examples — I would suggest example in form of Python code, something like:
```
>>> nestedKeys('1.2.3')
['1', '2', '3']
```
Having less words it looks more illustrative for me.

Reply:
Makes sense. However, per our discussion yesterday - tests will be used as examples, so I'll remove examples I've made here.